src/graphql/useSubmitTxn.ts

```diff
@@ -278,6 +281,43 @@ export default () => {
     [l1Network, submitHeliumLedger],
   )
 
+  const submitMintDataCredits = useCallback(
+    async (hntAmount: number) => {
+      if (!currentAccount || !anchorProvider) {
+        throw new Error('There must be an account selected to submit a txn')
+      }
+
+      dispatch(
+        sendMintDataCredits({
+          account: currentAccount,
+          anchorProvider,
+          cluster,
+          hntAmount,
+        }),
+      )
+    },
+    [anchorProvider, cluster, currentAccount, dispatch],
+  )
+
+  const submitDelegateDataCredits = useCallback(
+    async (delegateAddress: string, amount: number) => {
+      if (!currentAccount || !anchorProvider) {
+        throw new Error('There must be an account selected to submit a txn')
```

Comment:
Lets add this to the translation file as well

Reply:
Done!